src/algorithms/digi/PhotoMultiplierHitDigi.cc

```diff
@@ -90,10 +90,11 @@ eicrecon::PhotoMultiplierHitDigi::AlgorithmProcess(std::vector<const edm4hep::Si
             // pixel gap cuts
             // FIXME: generalize; this assumes the segmentation is `CartesianGridXY`
             auto id = ahit->getCellID();
-            dd4hep::Position pos_pixel, pos_hit;
+            dd4hep::Position pos_pixel, pos_hit, pos_hit_global;
             if(m_cfg.enablePixelGaps) {
               pos_pixel = get_sensor_local_position( id, m_cellid_converter->position(id) );
               pos_hit   = get_sensor_local_position( id, vec2pos(ahit->getPosition())     );
+              pos_hit_global =  m_cellid_converter->position(id);
```

Comment:
Save some time and call `m_cellid_converter->position(id)` once
```suggestion
              pos_hit_global = m_cellid_converter->position(id);
              pos_pixel = get_sensor_local_position( id, pos_hit_global );
              pos_hit   = get_sensor_local_position( id, vec2pos(ahit->getPosition())     );
```